srcpkgs/lsp-plugins/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'lsp-plugins'
+pkgname=lsp-plugins
+version=1.1.30
+revision=1
+build_style=gnu-makefile
+hostmakedepends="pkgconf php"
+makedepends="libsndfile-devel libX11-devel libglvnd-devel lv2 cairo-devel ladspa-sdk jack-devel"
+short_desc="Collection of free plugins compatible with LADSPA, LV2 and LinuxVST"
+maintainer="Artur Sinila <freesoftware@logarithmus.dev>"
+license="LGPL-3.0-or-later"
+homepage="https://lsp-plug.in/"
+distfiles="https://github.com/sadko4u/lsp-plugins/archive/refs/tags/${version}.tar.gz"
+checksum=9cf43257729093c240375b3640b1514dff34b092b83b54a5ee68d7e8565c8f80
+
+pre_build() {
+	CXXFLAGS="$CXXFLAGS -std=c++98"
```

Comment:
Where is this requirement documented?

Reply:
https://github.com/sadko4u/lsp-plugins/issues/191#issuecomment-812847630